package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "axios": "0.16.2",
-    "chart.js": "2.6.0",
+    "chart.js": "4.0.1",
```

Comment:
Why this specific version? ChartJS v4.0.1 was released 2 years ago. The latest version is v4.4.3. Does migrating to the latest version give you any trouble?